truelearn/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""major.minor.patch versioning convention
```

Comment:
Semantic Versioning? Perhaps we could just use "Semantic Versioning convention" here.